pylot/operator_creator.py

```diff
@@ -507,15 +507,17 @@ def add_lane_invasion_sensor(vehicle_id_stream):
     """
```

Comment:
Could you update the definition of the function to
`def add_lane_invasion_sensor(vehicle_id_stream: erdos.Stream) -> erdos.Stream:`